src/server.js

```diff
@@ -6,13 +6,22 @@ import app from './app.js'
 ;(async () => {
     let httpServer, httpsServer
     const appConfig = await config.fetch()
+    const HOST = appConfig.HOSTNAME || '0.0.0.0'
+    const PORT = appConfig.PORT || 3000
```

Comment:
Would it be easier to define these defaults in config.js, like you've done for the `TRANSLATE_ENDPOINT` variable?

https://github.com/DopplerHQ/yodaspeak/blob/e86b304d9032f1464bab9af331ad1aa611f98a35/src/config.js#L47

Reply:
It would be, but its easier to put them in `server.js` for now (until I refactor `config.js`) because of how configs can come from different sources, e.g. AWS Parameter or Secret Manager, as well as Doppler supplied env vars.